manim/mobject/mobject.py

```diff
@@ -2755,7 +2756,7 @@ def construct(self):
         return self
 
     # Errors
-    def throw_error_if_no_points(self) -> None:
+    def throw_error_if_no_points(self) -> None | Never:
```

Comment:
I don't think `Never` is used correctly here 🤔

Reply:
Then how should one type a function that is supposed to raise an `Exception`?

Reply:
Nice! I would love to continue contributing in typing but I'm still concerned about https://github.com/ManimCommunity/manim/pull/3086#issuecomment-1435777357. Unfortunately I'm missing time currently, but I'll see if I can still contribute a bit